src/context/game-state/provider.tsx

```diff
@@ -64,6 +64,10 @@ export const GameStateProvider = ({
     emit.iddqd({ animSpeed })();
   }, [animSpeed]);
 
+  useEffect(() => {
+    setShowPolicies(false);
+  }, [snap.context.policyMarket.active, setShowPolicies]);
+
```

Comment:
why this is needed? Seems like `false` is the default value

Reply:
This is so that the policy screen automatically closes when activating a card